dlp/src/main/java/dlp/snippets/InspectImageFile.java

```diff
@@ -48,13 +52,9 @@ public static void inspectImageFile(String projectId, String filePath) {
 
       // Specify the type and content to be inspected.
       ByteString fileBytes = ByteString.readFrom(new FileInputStream(filePath));
-      ByteContentItem byteItem = ByteContentItem.newBuilder()
-          .setType(BytesType.IMAGE)
-          .setData(fileBytes)
-          .build();
-      ContentItem item = ContentItem.newBuilder()
-          .setByteItem(byteItem)
-          .build();
+      ByteContentItem byteItem =
```

Comment:
I prefer the line by line over a single line (easier to understand).

Reply:
I agree - these changes were done unintentionally by `google-java-format`. I'll revert when I clean this up.